clang-tools-extra/clangd/index/SymbolCollector.cpp

```diff
@@ -576,6 +578,30 @@ SymbolCollector::getRefContainer(const Decl *Enclosing,
   return Enclosing;
 }
 
+std::vector<CXXConstructorDecl *>
+SymbolCollector::findIndirectConstructors(const Decl *D) {
+  auto *FD = llvm::dyn_cast<clang::FunctionDecl>(D);
+  if (FD == nullptr || !FD->isTemplateInstantiation()) {
+    return {};
+  }
```

Comment:
nit: should use without braces (and everywhere else in your code):
```suggestion
  if (FD == nullptr || !FD->isTemplateInstantiation())
    return {};
```
https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements

AFAIK most of code around tries to follow this pattern.

Reply:
I see, will refactor it later, have to say though that I am not that big of a fan. Not that I do not get why it is in the llvm coding standard, but I have multiple times already fallen victim to the expectation of there being a brace and extending an if just to not get an expected outcome. I guess its just a matter of getting used to.

Edit: Also I think our coding guidelines at work forbit them without braces.